packages/studio/src/components/PreviewWithUseComponents.tsx

```diff
@@ -4,33 +4,45 @@ import PreviewPanel from "./PreviewPanel";
 import Highlighter from "./Highlighter";
 import IFramePortal from "./IFramePortal";
 import { CSSProperties } from "react";
+import { useState } from "react";
+import { Tooltip } from "react-tooltip";
 
 const inlineStyles: CSSProperties = {
-  transform: "translateY(-50px)",
-};
-
-const innerIframeStyles: CSSProperties = {
-  transform: "translateY(50px)",
+  overflow: "scroll",
```

Comment:
without this style, do we cut off the iframe at the screen height so there is no vertical scrolling?

Reply:
No. I don't think that style is doing anything, removed.